tests/autoyast/installation.pm

```diff
@@ -239,6 +239,11 @@ sub run {
         elsif (match_has_tag('autoyast-error')) {
             die 'Error detected during first stage of the installation';
         }
+        elsif (match_has_tag('nvidia-validation-failed')) {
+           # nvidia repositories are unstable and really not needed for anything
```

Comment:
I think record info with short comment could be here to make it more visible.

Reply:
why ? is unsupported thirdparty problem ...

Reply:
Or soft-fail with bsc#1144831

Reply:
I went for soft failure needle with this reference

Reply:
> why ? is unsupported thirdparty problem ..., maybe if we want some 'statistics'

At least to show that something non standard, but not critical happened.